target/linux/ath79/dts/qca9558_compex_wpj558-16m.dts

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "compex,wpj558-16m", "qca,qca9558";
+	model = "Compex WPJ558 (16MB flash)";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_sig4;
+		led-failsafe = &led_sig4;
+		led-running = &led_sig4;
+		led-upgrade = &led_sig4;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		sig1 {
+			label = "red:sig1";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		sig2 {
+			label = "yellow:sig2";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		sig3 {
+			label = "green:sig3";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+		};
+
```

Comment:
What are these LEDs typically used for?

Reply:
Do you mean link indicators for the ethernet ports? Because that would make much more sense to me. Link indicators are very useful. Please use the LEDs for that.

Reply:
@CodeFetch NO. Please don't just give random "advice" here based on what you might have liked.

LEDs should generally follow the behavior of OEM firmware _if_ possible and reasonable. Otherwise they are left undefined _by default_.

Reply:
The hardware guide call them "RSSI" LEDs, it means that depending on the wireless signal strength, some of the four LEDs light up. However I have just flashed the stock firmware (WPJ558-v6.2.12_b190718.img) and the four LEDs are turned off by default. The user has to configure them manually depending on its wishes via the Compex web interface.
@CodeFetch You see the equipment as a finished product whereas it is a reference/hacker board.
IMHO we must let them disabled like default OEM firmware behavior.

Reply:
Based on the information I got here, I'm fine with the current LED setup.